applications/xquare-user-application/src/pages/login.tsx

```
@@ -0,0 +1,163 @@
+            <Button_square type="submit" disabled={!username || !password}>
+              로그인
+            </Button_square>
+
+            <LinkRow>
+              <Typography size="2x" weight="regular" align="center">
+                XQUARE계정이 없으신가요?
+              </Typography>
+
+              <a href="" style={{ textDecoration: "none", height: "1.8rem" }}>
+                <Typography
+                  size="2x"
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)